src/modules/edit_remove.test.js

```diff
@@ -0,0 +1,29 @@
+import addTask from './add-task.js';
+import storedArray from './storage.js';
+import removeTask from './remove-task.js';
+import editTask from './edit-task.js';
+import clearCompletedTask from './clear-completed-task.js';
+
+describe('test addTask and removeTask', () => {
+  test('test add task function', () => {
+    addTask('first task');
+    addTask('second task');
+    addTask('third task');
+    expect(storedArray[0].description).toBe('first task');
+    expect(storedArray.length).toEqual(3);
+  });
+  test('test remove task function', () => {
```

Comment:
- [ ] [OPTIONAL] It would be better if you renamed the test description string to `test editing task` for example so it describe the test